test/unit/line-ripple/index.test.tsx

```diff
@@ -3,10 +3,15 @@ import * as td from 'testdouble';
 import {assert} from 'chai';
 import {shallow} from 'enzyme';
 import LineRipple from '../../../packages/line-ripple/index';
+import {MDCLineRippleAdapter} from '@material/line-ripple/adapter';
 import {coerceForTesting} from '../helpers/types';
 
 suite('LineRipple');
 
+function getAdapter(foundation: any): MDCLineRippleAdapter {
```

Comment:
any --> MDCLineRippleFoundation

Reply:
When I do this change, I get an error

`Argument of type 'import("/home/travis/build/material-components/material-components-web-react/packages/line-ripple/node_modules/@material/line-ripple/foundation").MDCLineRippleFoundation' is not assignable to parameter of type 'import("/home/travis/build/material-components/material-components-web-react/node_modules/@material/line-ripple/foundation").MDCLineRippleFoundation'.`

Looks like imports for testing and for the component are different. Do you know how to fix this error?